lib/blip_ds.dart

```diff
@@ -49,6 +49,7 @@ export 'src/widgets/texts/ds_button_text.widget.dart' show DSButtonText;
 export 'src/widgets/utils/ds_cached_network_image_view.widget.dart'
```

Comment:
I think this spacing should be the same as the one on the right when there is nothing before the title. What do you think?

<img width="356" alt="image" src="https://user-images.githubusercontent.com/753061/188148319-0f71069b-6856-4ab4-9994-3032e8b41726.png">

Reply:
in reality this happened when the actions of the appbar was not null, it tried to keep the title centralized, and that's why it increased the margin of the border,
to fix this I put the  `centerTitle: false`